ml-agents/mlagents/envs/environment.py

```diff
@@ -222,7 +223,7 @@ def __str__(self):
                                                    for k in self._resetParameters])) + '\n' + \
                '\n'.join([str(self._brains[b]) for b in self._brains])
 
-    def reset(self, config=None, train_mode=True) -> AllBrainInfo:
```

Comment:
Does this mean it will no longer be possible to switch between training and inference configurations at reset ?

Reply:
It does.  Is this a case it is important for us to support?  This seems to me like an uncommon enough case that it would be fine to create a new environment, but I'm not sure I fully understand it.

Reply:
Is it possible to not remove this feature ?

Reply:
I agree with Vince. No reason to remove this feature, which is useful for those using ML-Agents in the context of Jupyter notebooks or other interactive tutorials.

Reply:
Sure thing, will fix